source/builders.txt

```diff
@@ -0,0 +1,117 @@
+.. _kotlin-sync-builders:
+
+===============================
+Implement Builders Code Pattern
```

Comment:
Not sure this title makes sense - users aren't implementing builders we are consuming them.